tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/submariner-io/shipyard/tools
 
-go 1.20
+go 1.21
```

Comment:
Why was this bumped? Note that we specify 1.20 elsewhere.

Reply:
It seems like a dependency requires it. https://stackoverflow.com/questions/77079565/go-mod-tidy-will-auto-upgrade-go-version-in-go-mod

Reply:
Why wasn't this done/needed in devel? Or these CVE's haven't been applied there yet?

Reply:
Weird. If I use golang `go1.20.10` this doesn't happen. That seems to show the StackOverflow answer is wrong...

I'll try the set again with that go version.